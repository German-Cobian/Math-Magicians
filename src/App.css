*,
*::after,
*::before {
  box-sizing: border-box;
}

.nav-bar,
.navbar-container {
  width: 100%;
  padding: 10px 0;
}

.navbar-container {
  display: flex;
  flex-direction: row;
  justify-content: space-around;
}

.brand-name {
  font-weight: bold;
  font-size: 25px;
}

.nav-links {
  display: flex;
  flex-direction: row;
  list-style: none;
}

.nav-links li {
  margin-right: 50px;
}

.keyboard,
.calculator-body {
  display: flex;
  flex-direction: column;
}

.container {
  display: flex;
  padding-top: 10em;
  justify-content: center;
}

.calculator-body {
  width: 400px;
  border: 1.5px solid black;
}

.screen {
  display: flex;
  width: 100%;
  height: 70px;
  padding: 0 1em;
  align-items: center;
  justify-content: flex-end;
  color: hsl(212, 1%, 100%);
  background-color: rgb(63, 60, 60);
}

.keyboard {
  width: 100%;
  background-color: hsl(210, 12%, 84%);
}

.row {
  display: flex;
  width: 100%;
}

.btn {
  width: 24%;
  border: none;
  height: 70px;
  margin: 0.5% 0.5%;
}

.btn:last-child {
  background-color: hsl(27, 78%, 57%);
}

.last-row > .btn:first-child {
  width: 49%;
}
