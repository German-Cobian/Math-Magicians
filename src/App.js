import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import './App.css';
import Navbar from './components/Navbar';
import CalculatorPage from './pages/CalculatorPage';
import Quote from './pages/QuotePage';
import Home from './pages/HomePage';

const App = () => (
  <>
    <Router>
      <Navbar />
      <Switch>
        <div className="container">
          <Route path="/calculator">
            <CalculatorPage />
          </Route>
          <Route path="/quote">
            <Quote />
          </Route>
          <Route exact path="/">
            <Home />
          </Route>
        </div>
      </Switch>
    </Router>
  </>
);

export default App;
