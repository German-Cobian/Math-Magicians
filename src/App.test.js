import renderer from 'react-test-renderer';
import App from './App';

describe('APP(main component)', () => {
  it('it should match snapshot', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
