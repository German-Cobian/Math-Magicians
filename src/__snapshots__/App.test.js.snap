// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP(main component) it should match snapshot 1`] = `
Array [
  <nav
    className="nav-bar"
  >
    <div
      className="navbar-container"
    >
      <span
        className="brand-name"
      >
        Math Magicians
      </span>
      <ul
        className="nav-links"
      >
        <li>
          <a
            aria-current="page"
            className="active-link"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Home
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </div>
  </nav>,
  <div
    className="container"
    computedMatch={
      Object {
        "isExact": true,
        "params": Object {},
        "path": "/",
        "url": "/",
      }
    }
    location={
      Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      }
    }
  >
    <div
      className="homepageContainer"
    >
      <h1
        className="title"
      >
        Welcome to our page!
      </h1>
      <div
        className="paragraphs"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
        incididunt ut labore et dolore magna aliqua. 
        Ut enim ad minim veniam, quis nostrud exercitation ullamco 
        laboris nisi ut aliquip ex ea commodo consequat. 
        Duis aute irure dolor in reprehenderit in voluptate velit 
        esse cillum dolore eu fugiat nulla pariatur. 
        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui 
        officia deserunt mollit anim id est laborum.
        </p>
        <p>
          Dictum non consectetur a erat nam at lectus. 
        In ante metus dictum at tempor commodo. 
        Quisque id diam vel quam elementum pulvinar 
        etiam non. Interdum velit euismod in 
        pellentesque massa placerat duis ultricies. 
        Erat velit scelerisque in dictum non. Tellus rutrum 
        tellus pellentesque eu tincidunt tortor aliquam. 
        Fringilla ut morbi tincidunt augue interdum velit. 
        A condimentum vitae sapien pellentesque 
        habitant morbi tristique senectus et. 
        Nunc aliquet bibendum enim facilisis gravida neque convallis a cras. 
        Lorem ipsum dolor sit amet consectetur. Integer malesuada 
        nunc vel risus commodo viverra maecenas accumsan lacus. 
        Arcu cursus euismod quis viverra nibh cras pulvinar.
        </p>
      </div>
    </div>
  </div>,
]
`;
